swagger.yaml

```diff
@@ -222,6 +222,16 @@ definitions:
       data:
         description: Configuration for data transfer
         $ref: '#/definitions/data-configuration'
+      imageRepositories:
+        description: Image repositories configuration
+        $ref: '#/definitions/image-repositories'
+
+  image-repositories:
+    type: object
+    properties:
+      authFile:
+        type: string
+        description: Image repositories authfile
```

Comment:
Can you add a link to the expected content of this file or to the an official documentation if available ?

Reply:
Updated.